src/subclass/container.rs

```diff
@@ -177,9 +177,9 @@ unsafe extern "C" fn container_set_focus_child<T: ObjectSubclass>(
     let instance = &*(ptr as *mut T::Instance);
     let imp = instance.get_impl();
     let wrap: Container = from_glib_borrow(ptr);
-    let widget: Widget = from_glib_borrow(wdgtptr);
+    let widget: Option<Widget> = from_glib_borrow(wdgtptr);
 
-    imp.set_focus_child(&wrap, &widget)
+    imp.set_focus_child(&wrap, widget)
```

Comment:
```suggestion
    imp.set_focus_child(&wrap, widget.as_ref())
```

We must never pass owned references to a `from_glib_borrow()` return value anywhere